common/src/main/java/com/datastrato/gravitino/Config.java

```diff
@@ -33,13 +33,13 @@ public abstract class Config {
   private final ConcurrentMap<String, String> configMap;
 
   private final Map<String, DeprecatedConfig> deprecatedConfigMap;
-  // Constant Array to hold all deprecated configurations
+  // Constant Array to hold all deprecated configuration keys, when a configuration is deprecated,
+  // we should add it here.
   private final DeprecatedConfig[] deprecatedConfigs = {
-    // Example deprecated configuration
-    //    new DeprecatedConfig(
-    //        "gravitino.test.string",
-    //        "1.0",
-    //        "Please use gravitino.test.string1 instead."),
+    new DeprecatedConfig(
```

Comment:
Now the code of config option is ok for me. 
It's better that we add the comment to tell users in which version this config option will be removed.
For the better maintain, this can be add the parameters for `DeprecatedConfig` instead of the comment.
For my view, we should remove this before we release 1.0 version.

Reply:
For this PR, we can keep the current changes, is it ok?

Reply:
Ok for me.